cvmfs/cvmfs.cc

```diff
@@ -1520,11 +1517,11 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
   } else if (attr == "user.maxfd") {
     attribute_value = StringifyInt(max_open_files_ - kNumReservedFd);
   } else if (attr == "user.usedfd") {
-    attribute_value = StringifyInt(atomic_read32(&open_files_));
+    attribute_value = no_open_files_->Print();
```

Comment:
At the moment, `Print()` does the right thing.  However, `Print()` in cvmfs is used for human-readable output, so the exact format of the string might change in the future.  Please add a function `ToString()` to the `perf::Counter` class, which transforms the number into a string.